utils/s3/client.cc

```diff
@@ -223,6 +232,64 @@ future<client::stats> client::get_object_stats(sstring object_name) {
     co_return st;
 }
 
+future<client::TagSet> client::get_object_tagging(sstring object_name) {
+    // see https://docs.aws.amazon.com/AmazonS3/latest/API/API_GetObjectTagging.html
+    auto req = http::request::make("GET", _host, fmt::format("{}?tagging", object_name));
```

Comment:
Use `req.query_parameters["tagging"]` = "" instead

Reply:
got it.